rust_team_data/src/v2.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+pub static BASE_URL: &str = "https://team-api.infra.rust-lang.org/v2";
```

Comment:
I'm about to ask something perhaps out of scope in this PR.

Can we please move `BASE_URL` out of the code and into an external env var?

The reason is: when I work on the triagebot and I use my own [test repository](https://github.com/apiraino/test-triagebot), I need to live patch URL to include other users so I can test f.e. PR review assignment.

Reply:
seems like that env var should live in triagebot though, no? 🤔

Reply:
Let's discuss that in a separate PR.